erdkotlin/src/main/java/com/elrond/erdkotlin/domain/transaction/models/TransactionToEstimate.kt

```diff
@@ -0,0 +1,13 @@
+package com.elrond.erdkotlin.domain.transaction.models
+
+import com.elrond.erdkotlin.domain.wallet.models.Address
+import java.math.BigInteger
+
+data class TransactionToEstimate(
```

Comment:
why do you need a separate class for this? because the `/transaction/cost` (and also the `/transaction/simulate` endpoint) expect the same thing

Reply:
Good catch.
Done 👍